README.md

```diff
@@ -41,17 +49,22 @@ python src/parse_clean_markdown.py --in data/markdown_raw --out data/markdown_pr
 
 This command will process Markdown files located in the `data/markdown_raw` directory and save the processed files to the `data/markdown_processed` directory.
 
-### Step 2: Set up OpenAI API key
 
-Create a `.env` file with a valid OpenAI API key:
+### Analysis
 
-```text
-OPENAI_API_KEY=<your-openai-api-key>
-```
+#### Embeddings :
 
-> Remark: The `.env` file is ignored by git.
+Run the Jupyter notebook `src/analysis_embeddings.ipynb` to visualize embeddings in a 2D and 3D.
 
-### Step 4: Create Chroma DB
+
+#### Chunk size
```

Comment:
To run this notebook successfully, you'll need to have created the various chroma databases of different sizes.

Wouldn't it be better to put "### Create Chroma DB" before?

Reply:
Oh yes. Could you change this after the PR?